tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -48,25 +49,35 @@
 default_test_quantile = 0.1
 
 
+REF_TYPE = List[Union[float, "REF_TYPE"]]
```

Comment:
I'm surprised that this kind of recursive type definition even works. Anyway, if not for anything else but for consistence, do you think it would be easier to have your ref values as np.ndarray, which would also help with typing?